docs/src/development/cli/_index.md

```diff
@@ -14,52 +14,24 @@ Its source code is hosted on [GitHub](https://github.com/platformsh/platformsh-c
 
 ## Before you begin
 
-You need to have:
-
-* PHP 5.5.9+
-* Git
-* A Bash-like shell
-
-  On Windows, the best way to get Bash is through [Windows Subsystem for Linux](https://msdn.microsoft.com/en-gb/commandline/wsl/about).
-  You can also use another Bash-compatible shell such as [Git Bash](https://gitforwindows.org/),
-  which you might have installed with git.
-
-For full requirements, see the [requirements on GitHub](https://github.com/platformsh/platformsh-cli#user-content-requirements).
+{{% cli-installation-requirements %}}
 
 ## 1. Install
 
-Install the CLI using this command:
-
-```bash
-curl -fsS https://platform.sh/cli/installer | php
-```
-
-If you run into issues or want to install manually, see the [installation instructions on GitHub](https://github.com/platformsh/platformsh-cli#user-content-installation).
+{{% cli-installation %}}
 
 ## 2. Authenticate
 
-To see and manage projects, you need to be authenticated with Platform.sh
-Run this command:
-
-```bash
-platform
-```
-
-You are asked to log in via a browser.
-This process creates SSH certificates on your computer for secure connections.
-(See more about [SSH connections](/development/ssh/_index.md).)
-
-Once you are logged in, a list of your projects appears, along with some tips for getting started.
+To see and manage projects, you need to be authenticated with Platform.sh.
 
-If you experience authentication issues or want to force a login, run the command `platform login`.
+{{% cli-installation-check %}}
 
 ## 3. Use
 
 Now you can run actions on your projects such as branching and merging.
 You can also simulate a local build of your codebase as if you were pushing a change to Platform.sh,
 including your services and data.
 
-To see all available commands, run `platform list`.
 To get more information on a particular command, preface that command with `help`:
```

Comment:
:lady_beetle: 
This no longer makes sense with the first sentence taken out ("that command" doesn't have an antecedent)

Reply:
Should be clearer now :)